packages/Webkul/Shop/src/Resources/lang/en/app.php

```diff
@@ -202,6 +202,32 @@
                         'shipping-handling'  => 'Shipping & Handling',
                         'tax-amount'         => 'Tax Amount',
                         'tax'                => 'Tax',
+                        'invoice-pdf' => [
```

Comment:
Please add a new line space here, and arrange the keys in alphabetical order based on the first letter of each key.